### DISCUSSION TITLE
如何快速学习新包的使用

### DISCUSSION THREAD

**Discussion by phdges** [note286/xduts#160] [Q&A] [answered]
请教一下各位都是在哪里迅速学习一个新的\usepackage使用方法的，尝试直接google条目很多，向chatgpt描述问题也较费时

1. 有没有迅速定位接口参数，给出简单使用用例的网站
2. 包的文档的官方地址一般都在哪里，去哪里找到README [CTAN官网](https://ctan.org/pkg/)好难用啊

**Comment by sikouhjw** [comment, marked as answer]
>     1. 有没有迅速定位接口参数，给出简单使用用例的网站

没有

>     2. 包的文档的官方地址一般都在哪里，去哪里找到README [CTAN官网](https://ctan.org/pkg/)好难用啊

ctan，如果你本地安装了 texlive，直接 `texdoc <package name>`
